main.py

```diff
@@ -35,11 +37,13 @@ def reEmail(fname):
 #You should use a ( and ) regex like " (extract this) " to extract a portion of the match
 
 def reAward(fname):
-  fhand = open(fname, 'r')
-  for line in fhand:
-    extr = re.findall('.*', line)
-    if len(extr):
-      print(extr)
+	fhand = open(fname, 'r')
+	for line in fhand:
+		extr = re.findall('.*Award *-(.*)', line)
```

Comment:
You extract an extra space before your award winner.

`extr = re.findall('- (.*)', line)`